pyart/graph/gridmapdisplay.py

```diff
@@ -62,9 +64,8 @@ def __init__(self, grid, debug=False):
         # set up the projection
         lat0 = grid.axes['lat']['data'][0]
         lon0 = grid.axes['lon']['data'][0]
-        self.proj = pyproj.Proj(proj='lcc', datum='NAD83',
-                                lat_0=lat0, lon_0=lon0,
-                                x_0=0.0, y_0=0.0)
+        self.proj = pyproj.Proj(proj='aeqd', datum='NAD83',
```

Comment:
I has having "huge" error with lamber, so I changed the data projection to azimuthal equidistant